ARCs/arc-0033.md

```diff
@@ -0,0 +1,60 @@
+---
+arc: 33
+title: Becoming an xGov
+description: Explanation on how to become Expert Governors.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+xGov status gives a governor the right to vote on proposals raised by the community, to ratify the proposals as valid for final community vote, where said proposals pertain to spending a previously specified amount of Algo in a given epoch on particular initiatives.
+ 
+The purpose of this ARC is to define the standard by which xGov status is achieved.
+ 
+## Specification
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC-2119</a>.
+ 
+### What is an xGov
+Expert Governors (xGov) are decentralized decision makers who demonstrate enduring commitment, are knowledgeable of the realities of the Algorand community, and whose interests are aligned with the good of blockchain.
+ 
+xGov **SHOULD** understand the community's needs and align their interests with those of the Algorand ecosystem.
+xGov **MAY** propose measures for a vote.
+xGov **MAY** participate in the designation of proposals.
+ 
+### Requirement to become an xGov
+The Foundation suggests that to become an xGov an Algo holder **MUST**:
+- Participate in another reward-bearing AF program
+- Commit the rewards to an xGov DAO for a period
+ 
+The foundation will maintain lists of eligible reward-bearing programs, and a list of qualified DAOs.
+ 
+#### Example
+Alice participates in governance during Q1 2023
+the potential reward is 5 Algo, Alice:
+1) Commit 100 Algo in an AF program
+2) Express her will to participate in xGov
+3) Will not receive 5 Algo at the end of Q1 2023. They are instead locked in one big xGov pool for 1 year
+4) Will receive 5 Algo at Q1 2024
+ 
+## Rationale
+In this ARC we seek community input on issues such as, but not limited to:
+- The types of reward-bearing activities that **SHOULD** be included in the qualification list (ie. general governance)
+- Whether xGovs **MUST** vote on all measures to keep their status
+- Whether xGovs tokens **SHOULD** be sold/transferred or **SHOULD** be burned when exiting the xGov DAO
+- Whether xGovs tokens expire after a year or gain more voting power if the stake remains in the DAO
+- Whether there **SHOULD** be a minimum and a maximum amount of Algos required to become an xGov
+- Whether there be a maximum amount of Algos that can be locked by xGovs for the each period, limiting the total number of xGovs
```

Comment:
There should not be a total maximum Algo commitment in any circumstance. This is particularly true if there is no maximum amount for being an xGov. Otherwise, a few large entities could come in and reach the maximum total commit threshold early on, thereby locking out a vast majority of Algo holders. 

Even if there is a maximum amount for being an xGov, there should not be a total maximum that can be locked for xGov generally. Whales can split wallets and still achieve the same thing as before by committing multiple wallets just below the per wallet limit. Instead, I will once again say that participation nodes should be the weeding mechanism. In such a scenario, whales can still split those wallets to try to evade the per wallet cap, but that also means they need to add more nodes, which makes collusive behavior (even among a single entity) more difficult. There is no way to stop whales from splitting wallets to evade either a per wallet cap or a from taking advantage of a global cap freeze out retail an control the process. But, at least a node requirement plus a per wallet cap (without a global cap) means they make it harder on themselves and nominally increase decentralization if they want to play those games.

Reply:
> Instead, I will once again say that participation nodes should be the weeding mechanism. In such a scenario, whales can still split those wallets to try to evade the per wallet cap, but that also means they need to add more nodes, which makes collusive behavior (even among a single entity) more difficult.

This is not accurate, who participates in consensus are Algorand Accounts not Algorand Nodes. This means you can still have different Participating Accounts on the same Node.

Reply:
> This is not accurate, who participates in consensus are Algorand Accounts not Algorand Nodes. This means you can still have different Participating Accounts on the same Node.

Fair enough and that is good to know. However, I still don't think there should be a global maximum amount because if whales are the first in, they would lock out middle/small holders by causing the cap to get reached quickly.